src/Compilers/CSharp/Test/Emit/CodeGen/PatternTests.cs

```diff
@@ -2130,5 +2130,105 @@ static class C {
                 Diagnostic(ErrorCode.WRN_SwitchExpressionNotExhaustive, "switch").WithLocation(9, 38)
                 );
         }
+
+        [Fact, WorkItem(32774, "https://github.com/dotnet/roslyn/issues/32774")]
+        public void BadCode_32774()
```

Comment:
>BadCode_32774 [](start = 20, length = 13)

Consider renaming the test since the test should not generate bad code. Perhaps `MatchAfterEliminatedEvaluationNode`.